app/pages/withdraw.tsx

```diff
@@ -74,6 +74,7 @@ export function Withdraw({ flow, tipId }: WithdrawProps) {
           });
           if (result.ok) {
             toast.success("Funds withdrawn!", { duration: 5000 });
+            router.push(PageRoutes.dashboard);
```

Comment:
@escapedcat thanks for the PR!

Unfortunately this section only handles webln and manual withdrawals. Lnurl withdrawals are not covered in this case. The toast should be moved somewhere else so it covers all scenarios. There is a redirect a bit further down which covers the case where the user is a tippee and they get directed to the final journey page, however that is also slightly flawed in how it is implemented. 

This is a bit more involved than I thought, I will create a new PR and tag you in it for review.

Reply:
PR: https://github.com/lightsats/lightsats/pull/86